ui/app/services/control-group.js

```diff
@@ -71,7 +72,8 @@ export default Service.extend({
     if (this.get('version.isOSS')) {
       return null;
     }
-    let pathForUrl = url.replace('/v1/', '');
+    let pathForUrl = parseURL(url).pathname;
+    pathForUrl = pathForUrl.replace('/v1/', '');
```

Comment:
This was in the changeset before your change, but just wondering, is there ever any chance here of a url like:

```
/v1/kv-like-thing/v1/v1/v1/value
```

?

Reply:
Ignore me! 

> If pattern is a string, only the first occurrence will be replaced.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace

Reply:
it's possible the url would be something like that if the user included the string `v1` somewhere in their path name (for example `foo-path/v1/bar-path/v1/value`), but `replace` should only replace the first instance of `/v1/` which will always be the prefix from the api path. so i think we should be okay here. :)

Reply:
heh, guess we came to the same conclusion!